### PR TITLE
Pipe: fix PipeEnrichedStatement can't contain redirection info

### PR DESCRIPTION
**Pull request by DanielWang2035**
As title.

### CODE REVIEW

File: iotdb-core/datanode/src/main/java/org/apache/iotdb/db/queryengine/plan/planner/TreeModelPlanner.java [hunk 1]
```diff
@@ -46,12 +46,16 @@
...
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
 import java.util.ArrayList;
 import java.util.List;
 import java.util.concurrent.ExecutorService;
 import java.util.concurrent.ScheduledExecutorService;
 
 public class TreeModelPlanner implements IPlanner {
+  private static final Logger LOGGER = LoggerFactory.getLogger(TreeModelPlanner.class);
```
Comment (Caideyipi) [inline iotdb-core/datanode/src/main/java/org/apache/iotdb/db/queryengine/plan/planner/TreeModelPlanner.java:58]: This can be removed

**Comment by Caideyipi** [comment]
Can search to replace all the substitutable "getFullPath()" with "getDeviceId()"....

**Comment by Caideyipi** [comment]
Can search to replace all the substitutable "getFullPath()" with "getDeviceId()"....

**Review by SteveYurongSu — APPROVED** [review]
Good

File: iotdb-core/datanode/src/main/java/org/apache/iotdb/db/queryengine/plan/planner/TreeModelPlanner.java (hunk 1 above)
Comment (SteveYurongSu) [reply iotdb-core/datanode/src/main/java/org/apache/iotdb/db/queryengine/plan/planner/TreeModelPlanner.java:58]: Will be resolved in https://github.com/apache/iotdb/pull/12543/files

**Comment by SteveYurongSu** [comment]
> Can search to replace all the substitutable "getFullPath()" with "getDeviceId()"....

Good. Will also be solved in https://github.com/apache/iotdb/pull/12543/files

**Pull request merged**